mlxtend/frequent_patterns/association_rules.py

```diff
@@ -30,6 +33,9 @@
 
 def association_rules(
     df: pd.DataFrame,
+    df_or: pd.DataFrame,
```

Comment:
If I see it correctly, the DataFrame is only required if `null_values` is `True`, correct? In this case, we could also make this argument optional and set it to `None` by default for backwards compatibility. (And then if someone saids `null_values=True` we raise an error that `df_or` should be provided.

Perhaps we should also rename `df_or` to `df_orig`. The name `df_or` made be first think of a logical or like `df_and` and `df_or`. What do you think?